src/Router/__test__/buildClientApp.test.tsx

```diff
@@ -3,13 +3,18 @@ import React from "react"
 import { buildClientApp } from "../buildClientApp"
 
 describe("buildClientApp", () => {
-  const getWrapper = async (props = {}) => {
+  const getWrapper = async (initialRoute = "/") => {
     const { ClientApp } = await buildClientApp({
       historyProtocol: "memory",
+      initialRoute: initialRoute || "/",
```

Comment:
Would be slightly nicer if a test didn’t replicate behaviour of the actual code.